extension/src/Xliff/XLIFFDocument.ts

```diff
@@ -51,51 +50,49 @@ export class Xliff implements XliffDocumentInterface {
 
   static fromDocument(xmlDoc: Document): Xliff {
     const fileElement = xmlDoc.getElementsByTagName("file")[0];
-    let _datatype = fileElement.getAttributeNode("datatype")?.value; // 'xml';
-    _datatype = isNullOrUndefined(_datatype) ? "" : _datatype;
-    let _sourceLang = fileElement.getAttributeNode("source-language")?.value;
-    _sourceLang = isNullOrUndefined(_sourceLang) ? "" : _sourceLang;
-    let _targetLang = fileElement.getAttributeNode("target-language")?.value;
-    _targetLang = isNullOrUndefined(_targetLang) ? "" : _targetLang;
-    let _original = fileElement.getAttributeNode("original")?.value;
-    _original = isNullOrUndefined(_original) ? "" : _original;
+    const _datatype = fileElement.getAttributeNode("datatype")?.value ?? "";
+    const _sourceLang =
+      fileElement.getAttributeNode("source-language")?.value ?? "";
+    const _targetLang =
+      fileElement.getAttributeNode("target-language")?.value ?? "";
+    const _original = fileElement.getAttributeNode("original")?.value ?? "";
```

Comment:
Vary satisfying change :smile: I could do the same for 61, 65, 69, 73,  77. But that actually breaks the tests, since unassigned != `undefined`. Later on I'll look into that, assigned `undefined` feels better than unassigned :upside_down_face: